src/common/HTTPResponse.ts

```diff
@@ -76,8 +76,8 @@ export class HTTPResponse {
     });
 
     this._remoteAddress = {
-      ip: responsePayload.remoteIPAddress,
-      port: responsePayload.remotePort,
+      ip: responsePayload.remoteIPAddress || '',
```

Comment:
I think these might also be behavior changing since previously we were putting `undefined` for `ip` when it doesn't exist in `remoteIPAddress`; now we're putting empty string.

IMO we can do two things:
* If we're sure that `remoteIPAddress` exists, we can do an assertion here
* If we know that `remoteIPAddress` might be `undefined` and we're fine with it; we can update `_remoteAddress` typing to reflect this. WDYT?
* We can accept this behavior change and keep these default values.
WDYT?

Reply:
Let's change the API here, good point!